app/ember/src/server/framework-preset-babel-ember.ts

```diff
@@ -12,6 +12,7 @@ function precompileWithPlugins(string: string, options: any) {
   return precompile(string, precompileOptions);
 }
 
+// @ts-ignore
```

Comment:
`// @ts-expect-error` and why it's expected is usually of more help to ourselves in the future

Reply:
It doesn't cause a problem when I run the bootstrap locally, but here it goes wrong:
https://nx.app/runs/gNGRsV0uRo5

```
src/server/framework-preset-babel-ember.ts(16,17): error TS4058: Return type of exported function has or is using name 'InputSourceMap' from external module "/tmp/storybook/node_modules/@types/babel__core/index" but cannot be named.
TSFILE: /tmp/storybook/app/ember/dist/ts3.9/src/client/preview/globals.d.ts
TSFILE: /tmp/storybook/app/ember/dist/ts3.9/src/client/preview/types.d.ts
TSFILE: /tmp/storybook/app/ember/dist/ts3.9/src/client/preview/render.d.ts
TSFILE: /tmp/storybook/app/ember/dist/ts3.9/src/client/preview/index.d.ts
TSFILE: /tmp/storybook/app/ember/dist/ts3.9/src/client/index.d.ts
TSFILE: /tmp/storybook/app/ember/dist/ts3.9/src/server/options.d.ts
TSFILE: /tmp/storybook/app/ember/dist/ts3.9/src/server/build.d.ts
ERR! FAILED (ts) :  
ERR! FAILED to compile ts: @storybook/ember@6.3.0-alpha.28 
```